src/routes/api/webhooks/github.ts

```diff
@@ -0,0 +1,300 @@
+import * as crypto from 'node:crypto'
+import { MessageEmbed } from 'discord.js'
+
+function createReleaseMessage(payload) {
+  let embed = new MessageEmbed()
+  embed.setTitle(`[${payload.repository.full_name}] ${payload.release.name}`)
+  embed.setColor('#ff9900')
+  embed.setDescription(payload.release.body)
+  embed.setURL(payload.release.html_url)
+  embed.setAuthor({
+    name: payload.repository.full_name,
+    url: payload.release.author.html_url,
+    iconURL: payload.release.author.avatar_url,
+  })
+  return {
+    content: 'New Release!',
+    tts: false,
+    embeds: [embed],
+  }
+}
+
+// https://gist.github.com/stigok/57d075c1cf2a609cb758898c0b202428
+function verifyGithubWebhookEvent(payloadbody, signature256: string) {
+  const token = process.env.GITHUB_WEBHOOK_SECRET
+  const sig = Buffer.from(signature256 || '', 'utf8')
+  const hmac = crypto.createHmac('sha256', token)
+  const digest = Buffer.from(
+    'sha256' + '=' + hmac.update(JSON.stringify(payloadbody)).digest('hex'),
+    'utf8'
+  )
+  if (sig.length !== digest.length || !crypto.timingSafeEqual(digest, sig)) {
+    throw new Error('Invalid webhook event')
```

Comment:
Not super familiar with JS - does throwing an error here translate to a certain response to the HTTP request? If not, I would suggest try/catching this method and responding with a 400 status code as well as logging the error. This will help us in debugging and making sure all requests get a response.